crates/pallet-offences-subspace/Cargo.toml

```diff
@@ -25,7 +25,7 @@ sp-std = { version = "4.0.0-dev", default-features = false, git = "https://githu
 [dev-dependencies]
 sp-io = { version = "4.0.0-dev", git = "https://github.com/paritytech/substrate", rev = "26d69bcbe26f6b463e9374e1b1c54c3067fb6131" }
 sp-core = { version = "4.0.0-dev", git = "https://github.com/paritytech/substrate", rev = "26d69bcbe26f6b463e9374e1b1c54c3067fb6131" }
-schnorrkel = "0.10.1"
+schnorrkel = "0.9.1"
```

Comment:
why rollback?

Reply:
Substrate still uses `0.9.1` internally and some types didn't match due to different versions. This was the only place we used a newer version, so I brought it back to the same version used by Substrate.